### DISCUSSION TITLE
Email Authentication : Email Verification Should Be Optional

### DISCUSSION THREAD

**Discussion by tricaricom** [serverpod/serverpod#2769] [Q&A] [answered]
I have set up email authentication.   When I go to create a new account on the Flutter app, I get the error "Email already in use", but this is a brand new database with no users at all.  I do not need to do any type of email validation, and I think this is part of the problem.   If I have this function:

```
sendValidationEmail: (session, email, validationCode) async {
      print("sendValidationEmail: $email $validationCode");
      return false;
}
```

then I get the error "Email address already in use".  This also happens if I do not have a `sendValidationEmail` function in place.  

Can this be configured to not require email verification?

**Comment by realtec** [comment, marked as answer]
I couldn't find a way to disable email verification either.
But I found a workaround 

```
auth.AuthConfig(
      sendValidationEmail: (session, email, validationCode) async {
        // Complete the account immediately
        await auth.Emails.tryCreateAccount(
          session,
          email: email,
          verificationCode: validationCode,
        );

        return true;
      },
    )

**Reply by tricaricom** [reply]
Great solution, thanks!

**Reply by Jeyanandh-32** [reply]
is it works?

**Reply by tricaricom** [reply]
Yes!
